package/aamp/aamp.mk

```diff
@@ -9,7 +9,7 @@ AAMP_SITE_METHOD = git
 AAMP_SITE = https://github.com/rdkcmf/rdk-aamp
 AAMP_INSTALL_STAGING = YES
 
-AAMP_DEPENDENCIES = libcurl libdash libxml2 cjson aampabr gst1-plugins-base wpeframework
+AAMP_DEPENDENCIES = libcurl libdash libxml2 cjson aampabr
```

Comment:
@sebaszm gst1-plugins-base and wpeframework are added to build these package before aamp in clean build. So does this work in clean build with out these ?

Reply:
I'm not sure a this moment, will double check... but why would AAMP depend on WPEFramework?

Reply:
It has ocdm header dependencies from WEPFramerwork

Reply:
You're right, I'll bring the dependecies for the time being back, though I don't feel this is a proper solution as we end up with circular dependencies.

Reply:
Ok, which one is causing the circular dependency ?

Reply:
It seems that after rebasing the branch, buildroot no longer complains about circular dependencies. Nevermind, then!